dxr_derive/src/lib.rs

```diff
@@ -145,7 +145,7 @@ pub fn try_from_value(input: TokenStream) -> TokenStream {
 
     let impl_block = quote! {
         impl #impl_generics #dxr::TryFromValue for #name #ty_generics #where_clause {
-            fn try_from_value(value: &#dxr::Value) -> Result<#name #ty_generics, #dxr::DxrError> {
+            fn try_from_value(value: &#dxr::Value) -> std::result::Result<#name #ty_generics, #dxr::DxrError> {
```

Comment:
Can you replace this with `::std::result::Result`, similar to other `std` imports?